### ISSUE TITLE
Hyphenate data attributes

### ISSUE THREAD

**Issue by os0x** [k0kubun/hamlit#5]
```haml
%div{ data: { raw_src: '...' } }
```

## Expected
```html
<div data-raw-src='...'>
```

## Actual
```html
<div data-raw_src='...'>
```

See: http://say26.com/new-in-haml-3-2-hyphenated-data-attributes

**Comment by felixbuenemann** [comment]
@os0x Does it work if you use `data: { raw: { src: '…' } } }`?

**Comment by mikelieser** [comment]
@felixbuenemann it works

**Comment by k0kubun** [comment]
I think it is a useful feature. I implemented it in 61df7a7.
Thank you!

**Comment by felixbuenemann** [comment]
Yes, it's even required if you have eg. both foo *and* foo-bar attributes.
A colleague had that case in haml today.
